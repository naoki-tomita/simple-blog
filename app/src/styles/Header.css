.header {
  max-width: 920px;
  margin: auto;
  padding: 24px 0;
}

.title {
  display: inline;
  padding: 8px;
  border: solid 1px #ccc;
  box-shadow: 0 4px 4px #ddd;
  border-radius: 16px;
}
