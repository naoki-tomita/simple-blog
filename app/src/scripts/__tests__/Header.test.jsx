import React from 'react';
import renderer from 'react-test-renderer';
import { Header } from '../Header';

describe("Editor", () => {
  it("should render successful.", () => {
    expect(renderer.create(
      <Header />
    )).toMatchSnapshot();
  });
})
