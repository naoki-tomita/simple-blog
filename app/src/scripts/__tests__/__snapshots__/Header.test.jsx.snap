// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Editor should render successful. 1`] = `
<div
  className="header"
>
  <h1
    className="title"
  >
    Simple blog
  </h1>
</div>
`;
